feature/subscriber-attributes/src/main/java/com/revenuecat/purchases/subscriberattributes/SubscriberAttributesManager.kt

```diff
@@ -57,6 +57,8 @@ class SubscriberAttributesManager(
         completion: (() -> Unit)? = null
     ) {
         obtainingDeviceIdentifiersObservable.waitUntilIdle {
+            // We are filtering out blank user IDs to guard against old users that could potentially have
+            // stored attributes as a blank user ID. See https://github.com/RevenueCat/purchases-android/pull/755
```

Comment:
`protect against old users` -> `skip requests for attributes that might have been stored as blank user IDs on older versions of the SDK`